src/datacenters/v1/README.md

```diff
@@ -0,0 +1,216 @@
+# v1 datacenter schema
+
+The v1 datacenter schema is designed to allow operators to control resources in three
+different scopes:
+
+- Datacenter-scoped resources
+- Environment-scoped resources
+- Application-scoped resources
+
+By combining these three scopes, operators can create robust datacenter templates that
+can be used to host and integrate virtually any application in any style of cloud
+environment.
+
+## Variables
+
+Though most of each datacenter schema is self-contained, there are some cases where datacenters
+templates need additional input from users before they can be used to power a datacenter. These
+inputs are called variables:
+
+```yml
+variables:
+  account:
+    # Variables can be arcctl resources
+    type: arcctlAccount
+    description: The DigitalOcean account used to power the environment
+    provider: digitalocean
+```
+
+Declared variables can be referenced anywhere else in the datacenter template that you'd like
+using the expression syntax, `${{ variables.<variable-name> }}`.
+
+## Datacenter resources
+
+Datacenter scoped [resources](../../%40resources/) are those which get created and destroyed with the lifecycle of
+the datacenter itself. This scoping allows operators to create cloud resources that will be
+shared across all environments and applications hosted by the datacenter. This can be VPCs,
+kubernetes clusters, API gateways, and more utilities that you'd want to share across
+environments.
+
+```yaml
+resources:
+  vpc:
+    type: vpc
+    name: arcctl-datacenter
+    account: ${{ variables.doAccount }}
+    region: ${{ variables.region }}
+```
+
+Every resource must declare a `type` that matches one of the arcctl [resource types](../../%40resources/)
+as well as an `account` capable of creating said type of resource. Once declared, the outputs of said
+resource type can be referenced elsewhere in the datacenter schema using the expression syntax,
+`${{ resources.<resource-key>.<output-key> }}`.
+
+```yml
+resources:
+  vpc:
+    # ...same as above...
+
+  cluster:
+    type: kubernetesCluster
+    name: arcctl-datacenter
+    kubernetesVersion: 1.26.5-do.0
+    account: ${{ variables.doAccount }}
+    region: ${{ variables.region }}
```

Comment:
May be nice to either showcase the whole file or add region to the variables example above.